public/data/kabir0x17.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Kabir",
+  "type": "personal",
+  "bio": "Open Source and Cyber Security Enthusiast!",
+  "avatar": "https://github.com/kabir0x17.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kabir0x17",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/kabir0x17",
+      "icon": "twitter"
+    }
+  ]
+}
```

I think you should add an endline here. Without this endline, it could cause problems with some IDEs and configurations. 😅

```suggestion
}

```

---

> I think you should add an endline here. Without this endline, it could cause problems with some IDEs and configurations. 😅

done,
thanks for the correction😄.